test/python/WMCore_t/ACDC_t/DataCollectionService_t.py

```diff
@@ -343,7 +343,7 @@ def notestMergeFakeFiles(self):
                                     'run_number': 1}],
```

Comment:
Seangchan, this test is still not running in jenkins. Can you please update the method name from `notest...` to `test...`?